lntest/mock/secretkeyring.go

```diff
@@ -21,9 +24,11 @@ func (s *SecretKeyRing) DeriveNextKey(
 }
 
 // DeriveKey currently returns dummy values.
-func (s *SecretKeyRing) DeriveKey(
-	_ keychain.KeyLocator) (keychain.KeyDescriptor, error) {
-
+func (s *SecretKeyRing) DeriveKey(keyLoc keychain.KeyLocator) (keychain.KeyDescriptor,
```

Comment:
nit: break before `keyLoc` and add newline after multi-line function definition.